src/components/ASPage.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import ASPerformance from './ASPerformance';
+
+
+export class ASPage extends Component {
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    return (
+      <div>
+        {Object.keys(this.props.views).map(key => {
+          return (
+            <div key={key} className="row">
+              <div className="col-md-6 panel panel-default">
+                <ASPerformance view={this.props.views[key]} viewId={key}/>
```

Comment:
Why not `Object.values(this.props.views).map(view => {...})` on line 14?
 so later on line 18 we could do simply.
`<ASPerformance view={view} viewId={view.id}/>`